include/logging/log_msg.h

```diff
@@ -83,7 +83,7 @@ struct log_msg_generic_hdr {
 	u16_t reserved : 14;
 };
 
-_Static_assert(sizeof(struct log_msg_generic_hdr) == sizeof(u16_t),
+BUILD_ASSERT_MSG(sizeof(struct log_msg_generic_hdr) == sizeof(u16_t),
```

Comment:
Is my browser messed up or are all of these "UILD_ASSERT_MSG"

Reply:
missing a B, so much for submitting PRs while in meetings...

Reply:
are you using "Refined Github"?